build.sh

```diff
@@ -30,4 +30,6 @@ if [ "${GOOS}" = "freebsd" ]; then
 fi
 
 echo "Building docker2aci..."
+go vet ./pkg/...
```

Comment:
good question, personally I see those `go vet` commands actually part of the "compile" phase independent of any tests, but I can move it over. We have a problem with Go 1.6.3 though in Semaphore (see https://github.com/appc/docker2aci/pull/210#issuecomment-255387525).